### ISSUE TITLE
3 Points Gradient 

### ISSUE THREAD

**Issue by guilegaspar** [airbnb/lottie-web#1266]
**Tell us about your environment**

MacOS Sierra 10.13.6

Google Chrome 
Version 69.0.3497.100 (Official Build) (64-bit)

After Effects 
Version 15.1.1 (Build 12)

![image](https://user-images.githubusercontent.com/15689745/46180196-e2638280-c273-11e8-8f1e-58697408a98c.png)

One shape layer on my comp has a gradient fill, with 3 points 
The 3 points are white but the first point is 0% opacity, second 100% and third is 0% opacity. 
I was wondering if 3 points gradients are supported.

On the screenshot above, on the right is the preview from the demo.html file. (I added the pink stroke just for the shape reference)
It looks like it only considers the 2 first points on the linear gradient. 
Any ideas?

Thanks!

AE file - [3Gradient_v01.zip](https://github.com/airbnb/lottie-web/files/2426232/3Gradient_v01.zip)

**Comment by bodymovin** [comment]
Hi, have you tried saving the aep project before exporting?
gradient data is taken from the aep file itself.

**Comment by guilegaspar** [comment]
Hey!
Yes, I saved the file before exporting. 
I also tried to close and open the project and close and open AE as well.

**Comment by bodymovin** [comment]
Ok, I'll take a look at it

**Comment by guilegaspar** [comment]
Oh I just figured out.
I just changed the name of the group inside the shape layer.
Changed from Group 1 to anything and it works 🤔

**Comment by bodymovin** [comment]
Right, if two layers have exactly the same name / path to the gradient, the exporter can miss the gradient data.
